src/cleanvision/issue_managers/image_property.py

```diff
@@ -356,13 +356,10 @@ def get_scores(
     ) -> pd.DataFrame:
         super().get_scores(raw_scores, issue_type, **kwargs)
         assert raw_scores is not None
+        median_score = raw_scores[self.score_columns[0]].median()
         scores = pd.DataFrame(index=raw_scores.index)
         scores[get_score_colname(issue_type)] = raw_scores[self.score_columns[0]].apply(
-            lambda x: 1.0
-            / max(
-                x / raw_scores[self.score_columns[0]].median(),
-                raw_scores[self.score_columns[0]].median() / x,
-            )
+            lambda x: 1.0 / max(x / median_score, median_score / x)
```

Comment:
kind of odd to do the redundant division instead of just min( x / median, median / x) which to my understanding appears the same, but maybe I'm missing something

Reply:
See [benchmarks](https://github.com/cleanlab/cleanvision/pull/217#issuecomment-1698037300).